### ISSUE TITLE
Add aria-label to exchange search input

### ISSUE THREAD

**Issue by pettinarip** [ethereum/ethereum-org-website#5793]
**Describe the bug**
The select/search input doesn't have an aria-label description.

**To Reproduce**

1. Go to https://ethereum.org/en/get-eth/
2. Scroll down to the select input
![image](https://user-images.githubusercontent.com/468158/160434052-1172ea6f-1d1e-4ae8-a066-a81be57999cd.png)

**Expected behavior**

Add aria-label to the underlying input element.

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

**Comment by HursitTarcan** [comment]
I can fix this

**Comment by KaranKaira** [comment]
Hey @pettinarip , I would like to contribute on this.

**Comment by pettinarip** [comment]
Hey @HursitTarcan I'll assign this one to @KaranKaira since you are already working on 2 other PRs.

Thanks both for the help!

**Comment by KaranKaira** [comment]
@pettinarip , what should be the label-text?

**Comment by pettinarip** [comment]
@KaranKaira I would use the same text we are using as the placeholder, in this case, `Type where you live` and that is an already translated text 👍🏼

**Comment by KaranKaira** [comment]
@pettinarip , plz take a look at PR #5831
